src/popup/pages/Dashboard.vue

```diff
@@ -143,6 +145,12 @@ export default {
         name: MODAL_TRANSFER_SEND,
       });
     },
+    openMultisigTransactionProposalModal() {
+      this.$store.dispatch('modals/open', {
```

Comment:
if you can, useDispatch

Reply:
This is options API. No way of using composables.